app/src/main/java/com/triviagenai/triviagen/trivia/presentation/results/ResultsScreen.kt

```diff
@@ -43,15 +60,15 @@ fun ResultsScreen() {
                 contentAlignment = Alignment.Center
             ) {
                 CircularProgressIndicator(
-                    progress = score,
+                    progress = score / 25,
```

Comment:
Why is this 25 for?

Reply:
I can improve the calculation for the percentage. Right now there are 5 questions of 5 points each. I'll multiply size of list time points.

Reply:
Okay, I understand it now. It is a little confusing, so it should be changed or added a comment.